charts/horizon/charts/tektonci-resources/templates/horizon/build-task.yaml

```diff
@@ -56,6 +56,10 @@ spec:
       {{- with .Values.horizon.buildTask.timeout.imageStep }}
       timeout: {{.}}
       {{- end}}
+      {{- if .Values.horizon.buildTask.privileged.imageStep }}
```

Comment:
Why need this?

Reply:
if we eliminate the privileged flag, we need to do the following:
Devices: /dev/fuse is required to use fuse-overlayfs inside of the container, this option tells Podman on the host to add /dev/fuse to the container so that containerized Podman can use it.

It is better that the pod does not depend on the host machine.

![image](https://user-images.githubusercontent.com/62157058/228128150-c03bcecd-5aa8-4389-8d9c-92e4f4adf746.png)

ref: https://www.redhat.com/sysadmin/podman-inside-kubernetes